tsconfig.base.json

```diff
@@ -12,13 +12,17 @@
     "typeRoots": ["node_modules/@types", "./typings"],
     "baseUrl": ".",
     "paths": {
-      "@fluentui/set-version": ["packages/set-version/src/index.ts"],
       "@fluentui/react-conformance": ["packages/react-conformance/src/index.ts"],
       "@fluentui/react-utilities": ["packages/react-utilities/src/index.ts"],
       "@fluentui/react-make-styles": ["packages/react-make-styles/src/index.ts"],
+      "@fluentui/make-styles": ["packages/make-styles/src/index.ts"],
+      "@fluentui/react-provider": ["packages/react-provider/src/index.ts"],
+      "@fluentui/theme": ["packages/theme/src/index.ts"],
```

Comment:
```suggestion
```

Can we also delete a this one? If there is a dependency on it somewhere, please let me know and I will remove it 🔪

Reply:
ah right, this is getting out of hand heh. 

As of today I assume I can generate all convergence packages and their deps by checking package version -> `9`. Am I right?

Reply:
> Am I right?

Almost 😢 `@fluentui/react-menu` was not bumped to v9...